dev/webpack.config.js

```diff
@@ -0,0 +1,160 @@
+/* eslint import/no-commonjs: 0 */
+const path = require('path');
+const without = require('lodash/without');
+
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const HtmlWebpackPluging = require('html-webpack-plugin');
+const BabiliPlugin = require('babili-webpack-plugin');
+const AutoDllPlugin = require('autodll-webpack-plugin');
+const HappyPack = require('happypack');
+
+const __DEV__ = process.env.NODE_ENV === 'development';
+const clean = arr => arr.filter(item => item !== false);
+
+module.exports = {
+  devtool: __DEV__ ? 'cheap-eval-source-map' : 'cheap-module-source-map',
```

Comment:
I do a lot of debugging in chrome, and a good source map for dev is a requirement :) Can we do so that we have a mode in which this is a reality? (maybe `cheap-eval-source-map` is good enough I need to try, but if it's not it's a pain :D)